streamfunction/client.go

```diff
@@ -81,6 +79,7 @@ func (c *clientImpl) Pipe(handler func(rxstream rx.Stream) rx.Stream) {
 	fac := rx.NewFactory()
 
 	for {
+		// TODO: escape out of here, cause will enter endless loop if c.Session has been destroyed
```

Comment:
When `Session` is closed, shall we find a way graceful shutdown this goroutine in next version?